corehq/apps/export/views/list.py

```diff
@@ -1050,3 +1051,16 @@ def page_context(self):
             context['create_url'] = '#odataFeedLimitReachedModal'
             context['odata_feeds_over_limit'] = True
         return context
+
+
+@location_safe
+class CommCareAnalyticsListView(BaseProjectDataView):
```

Comment:
This should also use `toggle_required` so people without permission cam't access it directly.